scheduler-worker/src/test/java/gov/nasa/jpl/aerie/scheduler/worker/services/SchedulingDSLCompilationServiceTests.java

```diff
@@ -655,6 +661,7 @@ export default function() {
                       Map.entry("duration", new ProfileExpression<>(new DiscreteProfileFromDuration(new DurationLiteral(Duration.of(1, HOUR)))))
                   ))
               ),
+                  Optional.of(PersistentTimeAnchor.DISABLED),
```

Comment:
are we sure about the indentation of these new anchor members ? they seem off.

Reply:
Optional.of in line 664 was off. Fixed